src/tests/dbus-tests/test_20_LVM.py

```diff
@@ -66,6 +66,11 @@ def _rescan_lio_devices(self):
         if ret != 0:
             raise RuntimeError("Cannot rescan vdevs: %s", out)
         self.udev_settle()
+        # device names might have changed, need to find our vdevs again
+        ret, out = self.run_command(r"find /sys/devices -maxdepth 7 -path '*tcm_loop*/tcm_loop_adapter_*/block/sd*' -printf '/dev/%f\n'")
```

Comment:
`find`? We have [glob](https://docs.python.org/3/library/glob.html) in Python and we are already using it in other test cases :-)

Reply:
Ha :-)